language/move-prover/tests/sources/functional/invariants.exp

```diff
@@ -28,9 +28,9 @@ error:  This assertion might not hold.
 
 error:  This assertion might not hold.
 
-     ┌── tests/sources/functional/invariants.move:143:9 ───
+     ┌── tests/sources/functional/invariants.move:150:9 ───
      │
- 143 │         invariant x > 1;
+ 150 │         invariant y > 1;
```

Comment:
Do we understand why this has changed? 

We do have some non-determinism that when multiple conditions can fail, only one of them will be displayed, and which one is kind of non-deterministic. Can both old and new invariant fail here?

If so we may want to rewrite the test a little such that only one condition can fail, to make our tests less flaky. This is particular important as we are moving to make those tests run on CircleCi.

Reply:
I'm not sure why this has changed. Error should only occur at line 150, and not line 143 though.

Reply:
I meant I did re-generate the exp file after I saw that MIRAI bot failed here. I don't know why exp file showed it was expected to fail at line 143 before(which was wrong).

Reply:
Then something regressed. Can you please try to investigate? We can't merge without understanding what goes wrong here.

Reply:
Ok.